File: kotlin-sample-app/src/main/java/org/remdev/android/kotlinapp/annotations/POJO.java
package org.remdev.android.kotlinapp.annotations;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Marker.
 * <br>
 *     Just to show that object should be a Plain Old Java Object
 *     as depending on implementation it may be converted somehow
 *     (e.g. to JSON).
 */
@Retention(RetentionPolicy.SOURCE)
@Target({
        ElementType.TYPE,
        ElementType.FIELD,
        ElementType.PARAMETER
})
public @interface POJO {
}

File: commander/src/main/java/org/remdev/android/commander/annotations/POJO.kt
package org.remdev.android.commander.annotations


/**
 * Marker.
 * <br></br>
 * Just to show that object should be a Plain Old Java Object
 * as depending on implementation it may be converted somehow
 * (e.g. to JSON).
 */
@Retention(AnnotationRetention.SOURCE)
@Target(
        AnnotationTarget.CLASS,
        AnnotationTarget.FILE,
        AnnotationTarget.FIELD,
        AnnotationTarget.VALUE_PARAMETER
)
annotation class POJO
